oracle/cpi.yml

```diff
@@ -0,0 +1,109 @@
+---
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-oracle-cpi
+    url: file://((local_oracle_cpi_release)) 
+
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: file://((local_oracle_light_stemcell))
+
+# Configure instance shapes 
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    availability_domain: ((ad))
+    instance_shape: VM.Standard1.2
+
+# Configure VCN and subnet
+- type: replace
+  path: /networks/name=default/subnets/0/cloud_properties?
+  value:
+    vcn: ((vcn))
+    subnet_name: ((subnet))
+
+# Enable registry job
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: registry
+    release: bosh
+
+- type: replace
+  path: /instance_groups/name=bosh/properties/registry?
+  value:
+    address: ((internal_ip))
+    host: ((internal_ip))
+    db: # todo remove
+      host: 127.0.0.1
+      user: postgres
+      password: ((postgres_password))
+      database: bosh
+      adapter: postgres
+    http:
+      user: registry
+      password: ((registry_password))
+      port: 25777
+    username: registry
+    password: ((registry_password))
+    port: 25777
+
+# Oracle Cloud Infrastructure properties for BOSH
+- type: replace
+  path: /instance_groups/name=bosh/properties/oci?
+  value: &cpi_conf
+    user: ((user))
+    fingerprint: ((fingerprint))
+    tenancy: ((tenancy))
+    region: ((region))
+    compartment: ((compartment))
+    apikey: ((apikey))
+    cpikey: ((cpikeys.private_key))
+    cpiuser: ((provisioned_username))
```

Comment:
Is there a reason this is customizable from something other than `vcap`? Especially since `vcap` should always, already be created by the base stemcell image?

Sorry, some of this is getting into CPI and implementation questions rather than this particular PR. I'd appreciate learning more, though.

Reply:
This one is also a vestige of custom stemcell that has `ubuntu` and not `vcap` as the provisioned user.   I'll look into removing this one too.